start.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import tempfile
+import yaml
+from socket import getfqdn, socket
+
+from clusterdock.models import Cluster, Node
+from clusterdock.utils import wait_for_condition
+
+DEFAULT_NAMESPACE = 'clusterdock'
+
+logger = logging.getLogger('clusterdock.{}'.format(__name__))
+
+
+def main(args):
+    # Image name
+    image = '{}/{}/clusterdock:kafka-{}-{}'.format(args.registry,
```

Comment:
Change the image name from `clusterdock:kafka-{}-{}` to `topology_apache_kafka` so that we can push images to our public clusterdock repo on Docker Hub.

Reply:
Done.